warehouse/upload.go

```diff
@@ -1098,7 +1096,12 @@ func (job *UploadJobT) loadTable(tName string) (alteredSchema bool, err error) {
 		var errTotalCount error
 		totalBeforeLoad, errTotalCount = job.getTotalCount(tName)
 		if errTotalCount != nil {
-			pkgLogger.Errorf(`Error getting total count in table:%s before load: %v`, tName, errTotalCount)
+			pkgLogger.Warnw(fmt.Sprintf("total count in table: %s before loading with err: %s", tName, errTotalCount),
+				warehouseutils.SourceID, job.upload.SourceID,
+				warehouseutils.DestinationID, job.upload.DestinationID,
+				warehouseutils.DestinationType, job.upload.DestinationType,
+				warehouseutils.WorkspaceID, job.upload.WorkspaceID,
+			)
```

Comment:
We can keep all the details in separate fields, keeping the message clean.

This way we can easily filter the msg.

```suggestion
			pkgLogger.Warnw("total count in table before loading"
				warehouseutils.SourceID, job.upload.SourceID,
				warehouseutils.DestinationID, job.upload.DestinationID,
				warehouseutils.DestinationType, job.upload.DestinationType,
				warehouseutils.WorkspaceID, job.upload.WorkspaceID,
				"error", errTotalCount,
				"tableName", tName,
			)
```